card/src/main/java/com/adyen/checkout/card/internal/ui/model/InstallmentOptionParams.kt

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2023 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by atef on 24/2/2023.
+ */
+
+package com.adyen.checkout.card.internal.ui.model
+
+import android.os.Parcelable
+import com.adyen.checkout.card.CardBrand
+import com.adyen.checkout.card.InstallmentOptions
+import kotlinx.parcelize.Parcelize
+
+/**
+ * InstallmentOptionParams is used for defining the details of installment options.
+ *
+ * Note: All values specified in [values] must be greater than 1.
+ */
+internal sealed class InstallmentOptionParams : Parcelable {
+    abstract val values: List<Int>
+    abstract val includeRevolving: Boolean
+
+    companion object {
+        private const val STARTING_INSTALLMENT_VALUE = 2
```

Comment:
Same question as before, do we need this default value on the SDK?

Reply:
This was done on `InstallmentConfiguration` I just copied it. I guess we need it when using a constructor with a max value.

Reply:
we probably won't need it will remove it